float64 password
string nodeName
bool nodeState
